### DISCUSSION TITLE
Reorder rows [Oct 04]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4015] [Discord Threads]
> 🙋‍♂️ **L-Dixon**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### No simple or direct way. You can create an `Numerical` column, Number or Decimal, insert numbers to identify order & sort by that number.  
#  
> 🙋‍♂️ **L-Dixon**  
#### Thanks! yea that was my work around.  Would love to be able to simply drag and drop.  Thanks though!  
#  
> 🙋‍♂️ **emdax**  
#### I also think this feature would be great.  All of the tactile features which let you interact with data in friendly way are a huge win to feel at home in your data view.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1024361837904535634)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
